### PR TITLE
Component markdown

### PR DESCRIPTION
**Pull request by Katedam**
Component creates one li using a markdown Title. Snapshot included

### CODE REVIEW

File: src/components/markdown/MarkdownFile.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+export default function MarkdownFile({ markdownTitles }) {
+  console.log('thiss', markdownTitles);
+  const titleList = markdownTitles.map((title) => {
+    return <li key={title}>{title}</li>;
+  });
+  return (
+    <ul>{titleList}</ul>
+  );
+
```
Comment (acrankin) [inline src/components/markdown/MarkdownFile.js]:
```suggestion
```

File: src/components/markdown/Document.js [hunk 2]
```diff
@@ -4,11 +4,13 @@ import Editor from './Editor';
 import store from '../../store';
 import { getMarkdown } from '../../selectors/selectMarkdown'; 
 import { updateMarkdown } from '../../actions/actionUpdateMarkdown';
+import MarkdownFile from './MarkdownFile';
 import styles from './Document.css';
 
 export default class Document extends PureComponent {
   state = {
     markdown: '# Hi there',
+    markdownTitles: ['tab1', 'tab2'],
```
Comment (acrankin) [inline src/components/markdown/Document.js]: These titles should be coming from the store. To complete this ticket, you'll need to complete the ticket "Create container to save markdown files (4pts)". Can't really do one without the other.

File: src/components/markdown/MarkdownFile.js (hunk 1 above)
Comment (Katedam) [reply src/components/markdown/MarkdownFile.js]: done!

**Review by acrankin — DISMISSED** [review]
You're going to need the container for this component as well, the tabs should be clickable and change the active file.

**Comment by pickuppatterns** [comment]
@acrankin Great point on the clickable.  I wanted to keep this PR small & managable so we created a new card in the trello backlog.  Here is the link: https://trello.com/c/RGAz6IIp/11-be-able-to-switch-between-markdown-files-using-markdownnav

In the meantime the list of titles will temporarily not be visible.

File: src/reducers/index.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+import { combineReducers } from 'redux';
+import markdownList from './reducerMarkdownList';
+import document from './reducerDocument';
+
+export default combineReducers({
+  markdownList,
+  document
```
Comment (mehtaphysical) [inline src/reducers/index.js:7]: consider changing this prop. `document` already exists in browsers. Maybe call it `doc`?

https://trello.com/c/3dHxG1VF/12-use-rename-document-in-combinedreducer

**Review by mehtaphysical — APPROVED** [review]

**Pull request merged**